examples/jsm/nodes/materials/MeshStandardNodeMaterial.js

```diff
@@ -36,6 +43,50 @@ export default class MeshStandardNodeMaterial extends NodeMaterial {
 
 	}
 
+	build( builder ) {
+
+		const lightNode = this.lightNode || builder.lightNode; // use scene lights
+
+		let { colorNode, diffuseColorNode } = this.generateMain( builder );
+
+		diffuseColorNode = this.applyStandardMaterialProperties( builder, { colorNode, diffuseColorNode } );
+
+		this.generateLight( builder, diffuseColorNode, lightNode );
+
+	}
+
+	applyStandardMaterialProperties( builder, { colorNode, diffuseColorNode } ) {
```

Comment:
I think that `generateStandardMaterial()` is more in compliance with generateLight, generateMain...

Reply:
Maybe `generateStandardMaterialProperties`?

Reply:
I think that generateStandardMaterial() is better

Reply:
OK.